ydb/docs/en/core/administration/upgrade.md

```diff
@@ -24,20 +23,20 @@ All minor versions within a major version are compatible for updates. Major vers
 * X.Y.* → X.Y+2.*: Update is impossible, major versions are inconsistent.
 * X.Y.* → X.Y-2.*: Update is impossible, major versions are inconsistent.
 
-A list of available versions can be found on the [download page](https://ydb.tech/en/docs/downloads/).
-
-{% note info %}
+A list of available versions can be found on the [download page](https://ydb.tech/en/docs/downloads/). The YDB release policy is described in more details in the [Release management](../development/manage-releases.md) article of the YDB development documentation.
 
-**Examples of compatible versions:**
+{% note warning %}
 
-v.22.2.5	->	v.22.2.47: Update is possible.
-v.22.2.47	->	v.22.4.21: Update is possible.
-v.22.2.40	->	v.22.4.21: Update is impossible, first upgrade to the latest version (v.22.2.47).
-v.22.2.47	->	v.22.5.5: Update is impossible, upgrade to the next major version (v.22.4).*
+Also, in any case, you cannot roll back more than 2 major versions relative to the version that was deployed at least once. This is because such an old version may not know how to work with data on the disks that the newer version persisted.
 
 {% endnote %}
 
-The YDB release policy is decribed in more details in the [Release management](../development/manage-releases.md) article of the YDB development documentation.
+### Examples of version compatibility
+
+* v.22.2.5	->	v.22.2.47: Update is possible.
+* v.22.2.47	->	v.22.4.21: Update is possible.
```

Comment:
I don't know why, but the example has major version shift for 2 instead of 1...
Must be 
v.22.2.47 -> v.22.3.21